src/components/Shared/UserInfoList.vue

```diff
@@ -43,7 +43,7 @@
         </tr>
         <tr
           class="user-info__row"
-          v-if="balance"
+          v-if="balance && balance.sum !== '-'"
```

Comment:
Let's define a local constant (e.g., `BALANCE_HIDDEN = '-'`) to clarify the meaning of `'-'` within code.